apps/web/public/documentation/change-log/10-2025.md

```
@@ -1,6 +1,72 @@
-## October 2025 - CLI updates, new components and docs upgrade
+## October 2025 - Major Icons Migration, CLI updates, new components and docs upgrade
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)